pkg/reconciler/v1alpha1/service/service_test.go

```diff
@@ -479,7 +508,7 @@ func TestReconcile(t *testing.T) {
 			Object: svc("config-fails", "foo", WithRunLatestRollout, WithInitSvcConditions,
 				// When the Route is Ready, and the Configuration has failed,
```

Comment:
This comment no longer reflects the condition

Reply:
Done.